modules/rss/src/main/java/org/mule/module/rss/transformers/RssParser.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.module.rss.transformers;
+
+
+import static org.jdom2.Namespace.NO_NAMESPACE;
+
+import com.rometools.rome.feed.WireFeed;
+import com.rometools.rome.feed.rss.Content;
+import com.rometools.rome.feed.rss.Description;
+import com.rometools.rome.feed.rss.Guid;
+import com.rometools.rome.feed.rss.Item;
+import com.rometools.rome.feed.rss.Source;
+import com.rometools.rome.io.FeedException;
+import com.rometools.rome.io.WireFeedParser;
+
+import static com.rometools.rome.io.impl.DateParser.parseDate;
+
+import com.rometools.rome.io.impl.RSS20Parser;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Locale;
+
+import org.jdom2.Document;
+import org.jdom2.Element;
+import org.jdom2.Namespace;
+
+
+public class RssParser extends RSS20Parser implements WireFeedParser
+{
+
+    private List<Namespace> namespaces;
+
+    @Override
+    public String getType()
+    {
+        return "rss_2.0";
+    }
+
+    @Override
+    public boolean isMyType(Document document)
+    {
+        return true;
+    }
+
+    @Override
+    protected Namespace getRSSNamespace()
+    {
+        return NO_NAMESPACE;
+    }
+
+    @Override
+    public Item parseItem(final Element rssRoot, final Element itemElement, final Locale locale)
+    {
+
```

Comment:
Do not add empty lines just because it is free. Add them only when it makes the code easier to read or to separate field/method/etc. Do not add empty lines at the beginning of a method body